plugins/inputs/ping/ping.go

```diff
@@ -122,27 +122,25 @@ func (p *Ping) Gather(acc telegraf.Accumulator) error {
 		p.listenAddr = getAddr(p.Interface)
 	}
 
-	for _, ip := range p.Urls {
-		_, err := net.LookupHost(ip)
+	for _, host := range p.Urls {
+		_, err := net.LookupHost(host)
 		if err != nil {
-			acc.AddFields("ping", map[string]interface{}{"result_code": 1}, map[string]string{"ip": ip})
+			acc.AddFields("ping", map[string]interface{}{"result_code": 1}, map[string]string{"url": host})
```

Comment:
Changing the tag from ip to url is a user-facing change.  The readme says url is the only tag so maybe ip was incorrect in the first place?

Then again why are we calling it url when it's a host or ip?  For example, url can be scheme://host/path but ping doesn't use scheme or path.

Reply:
Now I see your comment about ip being a regression